pkg/api/validate/vm_test.go

```diff
@@ -0,0 +1,105 @@
+package validate
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func TestDiskSizeIsValid(t *testing.T) {
+	for _, tt := range []struct {
+		name          string
+		diskSize      int
+		desiredResult bool
+	}{
+		{
+			name:          "size is valid",
+			diskSize:      129,
+			desiredResult: true,
+		},
+		{
+			name:          "size is not valid",
+			diskSize:      127,
+			desiredResult: false,
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			result := DiskSizeIsValid(tt.diskSize)
+
+			if result != tt.desiredResult {
+				t.Errorf("Want %v, got %v", tt.desiredResult, result)
```

Comment:
nit: I think this should begin with lower case, but can be fixed in a refactor/follow up later